gusto/physics.py

```diff
@@ -104,6 +106,39 @@ def __init__(self, state):
 
     def apply(self):
         self.lim_cond_rate.interpolate()
+        self.theta.assign(self.theta_new.interpolate())
         self.water_v.assign(self.water_v_new.interpolate())
         self.water_c.assign(self.water_c_new.interpolate())
-        self.theta.assign(self.theta_new.interpolate())
+
+
+class Fallout(Physics):
+    """
+    The fallout process of hydrometeors.
+
+    :arg state :class: `.State.` object.
+    """
+
+    def __init__(self, state):
+        super(Fallout, self).__init__(state)
+
+        self.state = state
+        self.rain = state.fields('rain')
+
+        # function spaces
+        Vt = self.rain.function_space()
+        Vu = state.fields('u').function_space()
+
+        # introduce sedimentation rate
+        # for now assume all rain falls at terminal velocity
+        terminal_velocity = 10  # in m/s
+        self.v = state.fields("rainfall_velocity", Vu)
+        self.v.project(as_vector([0, -terminal_velocity]))
```

Comment:
No need to make a whole function for this, just do:

```python
self.v = as_vector([0, -terminal_velocity])
```

Reply:
Oh, but then looking at the code, the `update_ubar` function requires fields, so OK.